q2_humann3/plugin_setup.py

```diff
@@ -93,7 +94,24 @@
         "bowtie_database": "directory containing the bowtie2 database reference files",
```

Comment:
I would be more explicit here - if this is the dir containing the .bt2l files then I would say 'directory containing the bowtie2 reference database index files'.

Reply:
This is a good suggestion, although, we should probably open a separate PR for it as the code isn't related to the code in this PR